src/w3c/templates/headers.js

```diff
@@ -253,6 +259,75 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderFeedback(conf) {
+  if (!conf.github) {
+    const hint = docLink`Use the ${"[github]"} configuration option to add a link to a repository.`;
+    showError("A link to a public repository is required.", name, {
+      hint,
+    });
+  }
+
+  if (!conf.github && !conf.wgPublicList) return;
+  const definitions = [];
+
+  // Github feedback...
+  if (conf.github) {
+    const issuesURL = new URL("./issues/", conf.github.repoURL);
+    definitions.push(
+      html`<dd><a href="${issuesURL}">${issuesURL.href}</a></dd>`
+    );
+  }
```

Comment:
GitHub is not required but we expect everyone to be using it, right? For `data-tests`, `<rs-changelog>` etc.

Reply:
Pretty much, yeah.